packages/react-composites/src/composites/ChatComposite/ChatComposite.tsx

```diff
@@ -49,13 +49,16 @@ export type ChatCompositeOptions = {
    * @defaultValue true
    */
   errorBar?: boolean;
+  /* beta:start */
```

Comment:
![image](https://user-images.githubusercontent.com/11863655/142617059-5252eca6-8f33-4319-b8d1-7225e28ac754.png)
This is how it looks after stripping, property removed while comments remain - if we wanna remove comments, there could be more edge cases to deal with to avoid false-positive removal

Reply:
Since this is just intermediate product of the build, we should be good not to remove comments

Reply:
feels weird though but I don't feel that it's worse trying to remove the comments too, so let's keep it that way